tests/test_transforms/test_timestamp/test_holiday_transform.py

```diff
@@ -224,3 +318,19 @@ def test_save_load(example_tsds):
 def test_params_to_tune():
     transform = HolidayTransform()
     assert len(transform.params_to_tune()) == 0
+
+
+def test_bigger_than_day_w_mon(two_segments_w_mon: TSDataset):
```

Comment:
You do not need to check the `HolidayTransform` function here, but the `define_period` function. Give it the date and check that the results are correct.